plugin/src/main/java/com/denizenscript/denizen/scripts/commands/world/TickCommand.java

```diff
@@ -0,0 +1,95 @@
+package com.denizenscript.denizen.scripts.commands.world;
+
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsRuntimeException;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.generator.*;
+import org.bukkit.Bukkit;
+import org.bukkit.ServerTickManager;
+
+public class TickCommand extends AbstractCommand {
+
+    // <--[command]
+    // @Name tick
+    // @Syntax tick [rate/step/sprint/freeze/reset] (amount:<amount>) (cancel)
+    // @Required 1
+    // @Maximum 2
+    // @Short Controls the server's tick rate.
+    // @Group world
+    //
+    // @Description
+    // Controls the server's tick rate. Versions 1.20+ only.

```

Comment:
Version should specify `MC` - there are versions of Minecraft, of Denizen, etc. so specifying is good to disambiguate.